src/Toolkit/V.php

```diff
@@ -422,6 +422,17 @@ public static function __callStatic(string $method, array $arguments): bool
                V::min($value, 2) === true;
     },
 
+    /**
+     * Checks for a valid hex color
+     */
+    'hexColor' => function ($value): bool {
+        if (!is_string($value) || $value === '') {
+            return false;
+        }
+
+        return mb_ereg_match('^#?([0-9a-fA-F]{3}|[0-9a-fA-F]{6})$', $value);
```

Comment:
Two comments on the logic:

- What about 4-digit and 8-digit hex colors with alpha transparency (RGBA)?
- Since the check doesn't involve multibyte characters, I think `preg_match()` should suffice. And since it's only about hex digits, it can be implemented even simpler with:

```php
return ctype_xdigit($value) === true && in_array(strlen($value), [3, 4, 6, 8]) === true;
```

(ignoring the hash sign, which needs to be checked separately depending on the result of our other discussion)